backend/whisper_with_diarization_as_methods.py

```diff
@@ -309,120 +302,89 @@ def write_list_to_csv(list_of_csv_content: List[str], output_csv_path: str, outp
-#
-#     The return would be another csv file (with the path specified by parameter csv_file_path) which has
-#     the following 4 column names
-#          - Timestamps
-#          - Speaker No
-#          - Text[Orig_lang]
-#          - Text[Eng]
-#
-#     Preconditions:
-#         - The length of the Timestamps and Speaker No in the 2 CSV files are the same
-#         - Likewise, the content of these 2 columns are assumed to be the same for each entry
-#     """
-#     first_csv_file_as_df = pd.read_csv(orig_lang_csv_file)
-#     second_csv_file_as_df = pd.read_csv(eng_lang_csv_file)
-#     eng_column = second_csv_file_as_df["Text[Eng]"]
-#
-#     # Creating a new csv file with path denoted by parameter comb_lang_csv_file
-#     dframe1 = pd.DataFrame(first_csv_file_as_df["Timestamps"])
-#     data1 = pd.concat([dframe1, first_csv_file_as_df["Speaker No"]], axis=1)
-#     data2 = pd.concat([data1, first_csv_file_as_df["Text[Orig Lang]"]], axis=1)
-#     data3 = pd.concat([data2, eng_column], axis=1)
-#     data3.to_csv(comb_lang_csv_file, encoding='utf-8', index=False)
-
 def main(process_selected: str, input_file: str, to_english_selection: bool, model_size_selection: str, destination_selection: str, diarize_model):
     # Step 1: Defining input audio path + defining CSV Headers
     input_audio_path = input_file # Insert audio file name and extension here (extensions can include: .mp3, .wav)
-    output_csv_headers = [] # Insert your headers here by replacing values of empty strings. Eg: ["Timestamps", "Speaker No", "Text[Eng]"]
+
     if process_selected == "Transcription Only":
-        output_csv_headers = ["Timestamps", "Speaker No", "Text[Orig Lang]"]
-    else:
+        output_csv_headers = ["Timestamps", "Speaker No", "Text[Orig Lang]"] # Insert your headers here by replacing values of empty strings. Eg: ["Timestamps", "Speaker No", "Text[Eng]"]
+        output_format = "transcription"
+    elif process_selected == "Translation Only":
         output_csv_headers = ["Timestamps", "Speaker No", "Text[Eng]"]
-    #elif process_selected == "Translation Only":
-    #    output_csv_headers = ["Timestamps", "Speaker No", "Text[Eng]"]
-    #else:
-    #    output_csv_headers = ["Timestamps", "Speaker No", "Text[Orig Lang]", "Text[Eng]"]
-    the_date_time = str(datetime.datetime.now()).replace("-", "_").replace(" ", "_").replace(":", "_").replace(".", "_")
-    output_csv_path = destination_selection + "/" + the_date_time + "_" + "streetwhisperapp.csv"
-    translate_to_english = to_english_selection # True denotes that if audio file is not in english, you want to translate text to english. If False, text would be transcribed based on autodetected language from Whisper
+        output_format = "translation"
+    else: # reaching here means: process_selected == 'Transcription + Translation Only'
+        output_csv_headers = ["Timestamps", "Speaker No", "Text[Orig Lang]", "Text[Eng]"]
+        output_format = "transcribe&translate"
```

Comment:
Not too sure about having & in the output file name.

Reply:
What would you suggest as the string for `output_format`? I think `transcribe_and_translate` would be too long.

Reply:
Maybe `transcribe_translate`?